scene/gui/text_edit.cpp

```diff
@@ -1709,6 +1709,7 @@ void TextEdit::gui_input(const Ref<InputEvent> &p_gui_input) {
 				h_scroll->set_value(h_scroll->get_value() + (100 * mb->get_factor()));
 			}
 			if (mb->get_button_index() == MouseButton::LEFT) {
+				_push_current_op();
```

Comment:
Think these should be moved closer to the actions, so for example scrolling the minimap doesn't end the current action.

Reply:
Fixed. Moved it to right before the functions that set the caret line and column.